transform-messages/apps/transform-messages/.gitignore

```diff
@@ -0,0 +1 @@
+orders/
```

Comment:
As I mentioned in A221 (https://github.com/RedHatTraining/AD221/pull/185#discussion_r783752505), to keep consistency with other GEs, we should adjust the dir names so that we have the following structure:

```
- AD221-apps
  - pattern-filter
    - apps
       - transform-messages
    - solutions
       - transform-messages
```